### PR TITLE
Improve contribution docs and remove unused package script

### PR DESCRIPTION
**Pull request by masashi-sutou**
## Motivation

I would like to contribute to this project to support mono-repo features like # 997. However, there are some obstacles that are difficult to contribute to this project, so we'll fix them first.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "ignore": "5.2.0",
     "joi": "17.6.0",
     "lodash": "4.17.21",
+    "nodemon": "2.0.15",
```
[the remark this answers is not in the document]
Comment (masashi-sutou) [reply package.json]: @jetersen 

- Thank you for reviewing my PR. Fixed in [54c33d8](https://github.com/release-drafter/release-drafter/pull/1066/commits/54c33d80375c0dbfa822b4eac87ee82611a65a2f)

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-[ ! -x "$(command -v yarn)" ] || yarn lint-staged
+[ ! -x "$(command -v yarn)" ] || yarn lint
```
[the remark this answers is not in the document]
Comment (masashi-sutou) [reply .husky/pre-commit]: @jetersen 

- Thank you for pointing out my mistake. Revert [2e60f2c](https://github.com/release-drafter/release-drafter/pull/1066/commits/2e60f2caeb41778567c078d0eb251f39633350c2)

**Comment by jetersen** [comment]
License wise your allowed to use, copy, modify :) https://github.com/release-drafter/release-drafter/blob/master/LICENSE

File: README.md [hunk 3]
```diff
@@ -355,13 +355,13 @@ If you have Node v10+ installed locally, you can run the tests, and a local app,
...
 # Install dependencies
-yarn
+yarn install
 
 # Run the tests
-npm test
+yarn test
 
 # Run the app locally
-npm run dev
```
Comment (jetersen) [inline README.md:364]: This was removed 😅

Maybe remove or suggest `yarn test:watch`

Comment (masashi-sutou) [reply README.md:364]: @jetersen 

- Fixed in [fdbeae7](https://github.com/release-drafter/release-drafter/pull/1066/commits/fdbeae7af33c8c5f3daa65e925197adabb2baab8)

**Comment by masashi-sutou** [comment]
> Thanks for contribution, perhaps it is not obstacles but improvements to existing things? 😅

- Yes, I'm sorry that the explanation of my motive is difficult to understand.

> Might be useful to look at this fork [349214e...blast-hardcheese:master](https://github.com/release-drafter/release-drafter/compare/349214e6b3b889d8b333c012cc61a1f1753baf40...blast-hardcheese:master) as an alternative to #997
> 
> See #935 original issue.

- I see, I will check #935.

**Pull request merged**